bin/phpunit

```diff
@@ -0,0 +1 @@
+../vendor/phpunit/phpunit/phpunit
```

Comment:
This file shouldn't be in here, it should be ignored - a .gitignore will need to be added to the repo.